### ISSUE TITLE
User lists as labels

### ISSUE THREAD

**Issue by Bossett** [bluesky-social/social-app#3135]
**Is your feature request related to a problem? Please describe.**

There is limited utility to user lists outside driving feeds - with users having to navigate to a profile & list in order to see a feed-like list of posts from people on the list.

**Describe the solution you'd like**

Users should be able to subscribe to user lists, and it should behave as an 'opposite' to mute. When you subscribe to a list, it should add a highlight & label to posts indicating which lists they are present on (this UI may be extensible to muted users on multiple mute lists).

**Additional**

'Follow' is (roughly) analogous to 'Block', so a user list may have a 'follow' function that works like the 'block' function (i.e. follows in sync). So mute/block <-> label/follow.

**Comment by PSingletary** [comment]
To make sure I understand, this is more than being able to pin a User list and read those posts
It would be an integration into the following feed and be labled what user list it came from?

**Comment by Bossett** [comment]
Yeah - though now that we have labels I would describe it like this:

A user on a user list that I subscribe to should behave as if they were labelled with a unique label, with the list name as the label description.

So like how user lists give people a way to have 'kinda feeds' through the official app, they should be extensible to have 'kinda labels' the same way.
